learntools/embeddings/nbs/1-embeddings.ipynb

```diff
@@ -641,6 +661,7 @@
     "\n",
     "In short, an **embedding layer** maps each element in a set of discrete things (like words, users, or movies) to a dense vector of real numbers (its **embedding**). \n",
     "\n",
+    "#### DB: In the interest of brevity, I'd leave out this implementation detail paragraph\n",
```

Comment:
What about keeping notes like this, but putting them in infoboxes that make it clear that this is some extra optional/skippable technical detail?

Reply:
I tried doing this in a few places in the Python course. My usual formatting convention was to put them in a block quote, prefixed with **Aside:**. e.g. "Aside: example calls" in [the functions lesson](https://www.kaggle.com/colinmorris/functions-and-getting-help).

Not super high priority, but it would be neat to have some consistent styling for these across lessons. If we really wanted to make them unobtrusive, we could make them collapsed by default, and expandable on click.